public/docs/ts/latest/quickstart.jade

```diff
@@ -10,479 +10,471 @@ include ../../../_includes/_util-fns
-    `System` is the module loader (from the `system.js` library),
-    a tool that can `import` code;
-    remember the `import` statement in our `AppComponent`?
+:marked
+  ## See It Run!
 
-      We're also asking `system.js` to "transpile" (AKA "compile") our
-      TypeScript source code into JavaScript ... right here in the browser.<br/><br/>
+  Running the [live example](/resources/live-examples/quickstart/ts/plnkr.html)
+  is the quickest way to see an Angular 2 app come to life.
+  
+  Clicking that link fires up a browser, loads the sample in [plunker](http://plnkr.co/ "Plunker"),
+  and displays a simple message:
 
-    3. We note the `<my-app>` tag in the `<body>`.
-    That's the custom HTML element we identified in the `@Component` decoration
-    adorning our `AppComponent` class.
+figure.image-display
+  img(src='/resources/images/devguide/quickstart/my-first-app.png' alt="Output of quickstart app")
+:marked
+  Here is the file structure:
+
+code-example(format="").
+  angular2-quickstart
+  ├─ app
+  │   ├── app.component.ts
+  │   └── boot.ts
+  ├─ index.htm  
+  └─ license.md
+:marked
+  Functionally, it's an `index.html` and two TypeScript files in an `app/` folder.
+  We can handle that!
+  
+  Of course we won't build  many apps that only run in plunker. 
+  Let's follow a process that's closer to what we'd do in real life.
+  
+  1. Set up our development environment
+  1. Write the Angular root component for our main app
+  1. Bootstrap it to take control of the main web page
+  1. Write the main page (`index.html`)
+  
+.l-sub-section
+  :marked
+    We really can build the QuickStart from scratch in five minutes 
+    if we follow the instructions and ignore the commentary.
+    
+    Most of us will be interested in the "why" as well as the "how" and that will take longer.
+:marked
 
 .l-main-section
-  :marked
-    ## Run it!
-
-    We need a static file server to serve our application to the browser.
-
-  .l-sub-section
-    :marked
-      Don't have a static file server handy?  Let's install one of our favorites
-      called [live-server](https://www.npmjs.com/package/live-server "Live-server")
-      with the **npm package manager**.
-
-      Don't have npm?
-      [Get it now](https://docs.npmjs.com/getting-started/installing-node "Installing Node.js and updating npm")
-      because we're going to use it now and repeatedly throughout this documentation.
-
-      Once you have `npm` installed, open a terminal window and enter
-
-    pre.prettyprint.lang-bash
-      code npm install -g live-server
-
-  :marked
-    Open a terminal window and enter
+:marked
+  ## Development Environment
+  
+  We'll need a place to stand (the application project folder), some libraries, 
+  some TypeScript configuration and the TypeScript-aware editor of your choice.
+
+  ### Create a new project folder
+  
+code-example(format="").
+  mkdir angular2-quickstart
+  cd    angular2-quickstart
```

Comment:
Maybe ... but I thought it drove the point home. I'm leaving it this way until I hear more complaints ;-)